activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -339,15 +339,14 @@ def teardown
     end
 
     def test_migrate_receives_correct_env_vars
-      verbose, version = ENV["VERBOSE"], ENV["VERSION"]
+      version = ENV["VERSION"]
 
-      ENV["VERBOSE"] = "false"
```

Comment:
This test doesn't expect/assert that VERBOSE is `false`. This test assert that `ActiveRecord::Migrator::migrate` is receiving correct attributes inside the method `ActiveRecord::Tasks::DatabaseTasks::migrate`. The attributes related to `VERSION` value but not `VERBOSE`.